main.go

```diff
@@ -156,9 +162,15 @@ Available arguments:
 			log.Println("Language Description: ", langDesc)
 		}
 
+		if stack != nil {
+			log.Println("Stack:", *stack)
+		}
+
 		// Create a slug at Heroku
-		slug, err := c.SlugCreate(app, processTypes, &heroku.SlugCreateOpts{
-			Commit: &commit,
+		slug, err := svc.SlugCreate(context.TODO(), app, heroku.SlugCreateOpts{
+			Stack:        stack,
```

Comment:
This can just be `&stack`, just like `&commit` below it.  This then relies upon the JSON `omitEmpty` to avoid trying to configure something when not supplied.

Reply:
I believe JSON `omitEmpty` will send `""` if the string exists. I think I misunderstood how `flag.String` works w/r/t to `nil`s.